lib/logflare/sql.ex

```diff
@@ -530,6 +539,22 @@ defmodule Logflare.Sql do
     end
   end
 
+  defp validate_sandboxed_query_ast(ast, %{sandboxed_query: sandboxed_query} = data) do
+    case extract_replacement_query_from_ast(ast, data) do
+      {:ok, _replacement_query} ->
+        :ok
+
+      {:error, transformed_statements} ->
+        Logger.warning("Sandboxed query validation: would produce nil replacement_query",
+          error_string: sandboxed_query,
+          ast_statements_count: length(transformed_statements),
+          first_statement: List.first(transformed_statements)
```

Comment:
instead of adding more situational fields, lets just add it to the error message instead. also easier to search.
fine to leave the query on error_string

Reply:
also, as above, logging on each sql parse would lead to a lot of log lines. probably better to add as metadata for the raise.